### PR TITLE
LPS-83755 Update CSS Builder and Theme Builder to read artifacts from pom

### PR DESCRIPTION
**Pull request by gamerson**
Can you please ask Brian to publish `css-builder` and `portal-tools-theme-builder`?  Then we can update the project templates.

### CODE REVIEW

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:relevant

**Comment by lawrence-lee** [comment]
@Ithildir Can we also publish `gradle-plugins-css-builder` and a new `gradle-plugins`?

**Comment by lawrence-lee** [comment]
and a new `gradle-plugins-theme-builder`

**Comment by Ithildir** [comment]
Hi @petershin! Can you please review this pull? Thanks!!

File: modules/util/css-builder/src/main/java/com/liferay/css/builder/maven/BuildCSSMojo.java [hunk 1]
```diff
@@ -53,20 +52,24 @@
...
 
-				String artifactId = componentDependency.getArtifactId();
+			for (Dependency dependency : _project.getDependencies()) {
+				String artifactId = dependency.getArtifactId();
 
 				if (artifactId.equals("com.liferay.frontend.css.common") &&
 					(_cssBuilderArgs.getImportDir() == null)) {
 
-					Artifact artifact = _resolveArtifact(componentDependency);
+					artifact = _resolveArtifact(dependency);
```
Comment (Ithildir) [inline modules/util/css-builder/src/main/java/com/liferay/css/builder/maven/BuildCSSMojo.java]: Shouldn't you `break` once you find `artifact`?

**Review by Ithildir — CHANGES_REQUESTED** [review]
cc @petershin

**Review by petershin — CHANGES_REQUESTED** [review]

**Comment by gamerson** [comment]
Thanks for the review guys.  I've pushed fixes for all suggestions.

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:relevant

File: modules/util/css-builder/src/main/resources/META-INF/maven/com.liferay/com.liferay.css.builder/plugin-help.xml [hunk 2]
```diff
@@ -49,7 +49,7 @@
 				<parameter>
 					<name>docrootDirName</name>
 					<type>java.lang.String</type>
-					<deprecated>As of 2.1.0, replaced by {@link #setBaseDir(File)}</deprecated>
+					<deprecated>As of Judson (7.1.x), replaced by {@link #setBaseDir(File)}</deprecated>
```
Comment (Ithildir) [inline modules/util/css-builder/src/main/resources/META-INF/maven/com.liferay/com.liferay.css.builder/plugin-help.xml]: @petershin, can you please try to figure out why this change happened? I think there's something wrong with SF or with the Javadoc Formatter 🤔 Also, can you revert these wrong changes made in `modules/sdk` and `modules/util`? Thanks!

Comment (petershin) [reply modules/util/css-builder/src/main/resources/META-INF/maven/com.liferay/com.liferay.css.builder/plugin-help.xml]: @Ithildir - I think its with the `Javadoc Formatter` will take a look

Comment (Ithildir) [reply modules/util/css-builder/src/main/resources/META-INF/maven/com.liferay/com.liferay.css.builder/plugin-help.xml]: I'm reverting them, if you want to work on something else in the meantime 😊

Comment (petershin) [reply modules/util/css-builder/src/main/resources/META-INF/maven/com.liferay/com.liferay.css.builder/plugin-help.xml]: Appreciate it. Will send a patch for this issue

**Comment by Ithildir** [comment]
Sent to https://github.com/brianchandotcom/liferay-portal/pull/61852.
